github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -455,7 +455,14 @@ presubmits:
         - "/usr/local/bin/runner.sh"
         - "/bin/sh"
         - "-c"
-        - "apt update && apt install gettext-base -y && export TARGET=kind-k8s-sriov-1.14.2 && automation/test.sh"
+        - >
+            apt update &&
+            apt install gettext-base -y &&
+            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz &&
```

Comment:
Would it make sense to install Go in the base image `gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9` instead? Ram did it in his https://github.com/kubevirt/project-infra/pull/373

Reply:
Didn't want to change image, can we use the kubevirt bootstram image here ?

Reply:
There is a different image for kubevirt with Go installed? I'd go for that.

Reply:
Why do we need gettext and all this?